components/mediation-monitor/mediation-data-publisher/org.wso2.carbon.prometheus.publisher/src/main/java/org/wso2/carbon/prometheus/publisher/service/MetricResource.java

```diff
@@ -42,11 +38,13 @@
 
     private static Log log = LogFactory.getLog(MetricResource.class);
     private MetricPublisher metricPublisher;
+    private boolean isPrometheusPublishingEnabled;
 
     public MetricResource(String urlTemplate) {
 
         super(urlTemplate);
         metricPublisher = new MetricPublisher();
+        isPrometheusPublishingEnabled = Boolean.parseBoolean(System.getProperty("enablePrometheusBE"));
```

Comment:
shall we rename it to something else? something like enablePrometheusAPI? @a5anka whyt?

Reply:
Also, shall we move the parameter to a constant?

Reply:
Done in https://github.com/wso2/carbon-mediation/pull/1229/commits/ebd74f7c222d652980ff4b3ff59bd478c7d0509e